### ISSUE TITLE
Dalfox doesn't detect the Reflected parameters if is no there any whitespaces

### ISSUE THREAD

**Issue by knassar702** [hahwul/dalfox#354]
Hello @hahwul , I've notice that Dalfox doesn't detect `Dalfox` word for reflected parameters if is no there any whitespace between this word and the response body
for example this a simple web app that takes the url parameters and convert it to simple format like this `param1:hello,param2:hello3` if you tried to use Dalfox on this target it will not find any reflection 

```python
# $ pip install flask
from flask import Flask,request

app = Flask(__name__)

@app.route("/")
def index():
    name = request.args
    data = ""
    for k,v in name.items():
        data += f"{k}:{v},"
    return f"Hello {data}"

app.run(host='0.0.0.0')
```

### Test

```bash
$ curl localhost:5000/?name=khaled&testing=true
Hello name:khaled,testing:true,

root@hacking:~ # curl "http://localhost:5000/?name=<script>alert``</script>"
Hello name:<script>alert``</script>,

root@hacking:~ # echo "http://localhost:5000/?name=f" | ./dalfox pipe

    _..._
  .' .::::.   __   _   _    ___ _ __ __
 :  :::::::: |  \ / \ | |  | __/ \\ V /
 :  :::::::: | o ) o || |_ | _( o )) (
 '. '::::::' |__/|_n_||___||_| \_//_n_\
   '-.::''    

🌙🦊 Powerful open source XSS scanning tool and parameter analyzer, utility

 🎯  Target                 Stdin (pipeline)
 🏁  Method                 GET
 🖥   Worker                 100
 🔦  BAV                    true
 ⛏   Mining                 true (Gf-Patterns)
 🔬  Mining-DOM             true (mining from DOM)
 ⏱   Timeout                10
 📤  FollowRedirect         false
 🕰   Started at             2022-02-21 21:14:02.814439577 +0000 UTC m=+0.030088910

 >>>>>>>>>>>>>>>>>>>>>>>>>
[*] 🦊 Start scan [SID:0][0/1][0.00%%] / URL: http://localhost:5000/?name=f
[I] Found 0 testing point in DOM base parameter mining
[I] Content-Type is text/html; charset=utf-8
[*] [duration: 2.689336927s][issues: 0] Finish Scan!

```

best regards
@knassar702

**Comment by hahwul** [comment]
Hi @knassar702 
Oh, I'm sorry to late. First, thank you for submit issue :D
I'll check it out!

**Comment by hahwul** [comment]
## Test
```
./dalfox url http://192.168.0.4:8000/\?name\=fz --skip-headless
```
![스크린샷 2022-03-24 오후 10 38 54](https://user-images.githubusercontent.com/13212227/159928516-49489086-ce45-4668-b2e7-9c2fea076e2d.png)

## Patch
v2.7.2 (hotfix)

**Comment by hahwul** [comment]
Hi @knassar702 
In case of indiscriminate reflection, there was a code to verify to reduce unnecessary scanning, but it seemed to decrease the detection rate, so I removed it. It will be released in `v2.7.2`. 

I didn't have anything special during the test, but if you have the same problem, please reopen it!
